src/ignitions.R

```diff
@@ -100,7 +100,7 @@ if(isDatasheetEmpty(IgnitionsPerIteration)) {
 
 if(isDatasheetEmpty(ResampleOption)) {
   updateRunLog("No Minimum Fire Size chosen.\nDefaulting to a Minimum Fire Size of 1ha and drawing 10% extra fires for resampling.\nPlease see the Fire Resampling Options table for more details.", type = "info")
```

Comment:
Should we update the run log statement to say "Default to no Minimum Fire Size and no resampling."?